airflow/providers/neo4j/example_dags/example_neo4j.py

```diff
@@ -23,13 +23,8 @@
 from airflow.providers.neo4j.operators.neo4j import Neo4jOperator
 from airflow.utils.dates import days_ago
 
-default_args = {
-    'owner': 'airflow',
-}
-
 dag = DAG(
     'example_neo4j',
-    default_args=default_args,
```

Comment:
I feel that we shouldn't remove the `owner` argument from the examples. We can however add it to the DAG argument. Like:
```
dag  = DAG(...,
                    default_args={'owner':'airflow'},
                    ...
)
```
This lets people know that such arg is possible

Reply:
Owner isn't really _used_ much anywhere anymore, with RBAC it doesn't have much significance anymore anyway.

But either way: if want to show owner we can show it _once_, not on every example dag :)

Reply:
We do also provide an example of setting the `owner` as part of `default_args` on the Concepts: DAGs documentation (https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html). Perhaps that is good enough @ashb?